src/platform/plugins/shared/controls/public/controls/data_controls/options_list_control/components/options_list_popover_action_bar.tsx

```diff
@@ -99,7 +138,39 @@ export const OptionsListPopoverActionBar = ({
           {allowExpensiveQueries && (
             <EuiFlexItem grow={false}>
               <EuiText size="xs" color="subdued" data-test-subj="optionsList-cardinality-label">
-                {OptionsListStrings.popover.getCardinalityLabel(totalCardinality)}
+                {OptionsListStrings.popover.getCardinalityLabel(totalCardinality)} |{' '}
+                <EuiToolTip
+                  content={
+                    hasTooManyOptions
+                      ? OptionsListStrings.popover.getMaximumBulkSelectionTooltip()
+                      : undefined
+                  }
+                >
```

Comment:
This tooltip isn't accessible via keyboard controls - but since it offers useful information, I think it should be?

![May-27-2025 15-30-11](https://github.com/user-attachments/assets/7e43f8c7-f948-444d-ba23-6c9fcfee3d36)

Reply:
Is that something EUI supports? I'm not sure we can focus on a disabled button, which is required to trigger the tooltip.

Reply:
Linking off to https://github.com/elastic/eui/issues/5666 here, since this currently isn't possible